python/cudf/_gdf.py

```diff
@@ -437,7 +437,11 @@ def hash_columns(columns, result):
     col_out = result.cffi_view
     ncols = len(col_input)
     hashfn = libgdf.GDF_HASH_MURMUR3
-    libgdf.gdf_hash(ncols, col_input, hashfn, col_out)
+    if initial_hash_values is None:
+        initial_hash_values = ffi.NULL
+    else:
+        initial_hash_values = unwrap_devary(initial_hash_values)
+    libgdf.gdf_hash(ncols, col_input, hashfn, initial_hash_values, col_out)
```

Comment:
Outside of the scope of this PR, but we should move this to Cython in the future.